docs/sdk/migrations/datil.md

```diff
@@ -0,0 +1,176 @@
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Migrating to Datil
+
+Previously Lit networks were powered by a pre-Bedrock Optimism L2 rollup called [Chronicle](../../network/rollup.mdx). In our effort to deliver a more performant and stable backend for Lit's infrastructure, we're launching a new network, Vesuvius, powered by [Arbitrum Orbit](https://arbitrum.io/orbit).
+
+Depending on the existing Lit network you are using, you should migrate to it's corresponding Datil network:
+
+| Currently Available | Requires Payment | Your Current Network | Network to Migrate to | Description                                                 |
+| --------------------| ---------------- | -------------------- | -------------------- | ------------------------------------------------------------ |
+|          ❌         |          ✅      | `habanero`           | `datil`               | Decentralized mainnet designed for production use cases      |
+|          ❌         |          ✅      | `manzano`            | `datil-test`          | Decentralized testnet designed for pre-production deployment |
+|          ✅         |          ❌      | `cayenne`            | `datil-dev`           | Centralized testnet designed for early-stage development     |
+
+Like their counterparts, `datil` and `datil-test` require payment for usage of the Lit network via [Capacity Credits](../capacity-credits.md); however, `datil-dev` does not.
+
+## Breaking Changes and Important Updates
+
+### Datil Resources
+
+Datil has a new [block explorer](https://vesuvius-explorer.litprotocol.com/), and Lit test tokens can be received from [this faucet](https://datil-dev-faucet.vercel.app/).
```

Comment:
'..., and a new Lit test token (testLIT) which can be received from [this faucet]'

Reply:
Addressed via [this commit](https://github.com/LIT-Protocol/docs/pull/306/commits/73f7fc652a19a5d1eefc635183091d1e4b91a9b1)